server/src/main/java/org/elasticsearch/cluster/RestoreInProgress.java

```diff
@@ -75,7 +76,7 @@ public int hashCode() {
 
     @Override
     public String toString() {
-        return new StringBuilder("RestoreInProgress[").append(entries).append("]").toString();
+        return Strings.toString(this);
```

Comment:
The entries didn't have a `toString` so the current `toString` was a little pointless, but we do have a perfectly fine `toXContent` anyway :)

Reply:
But this one is very verbose compared with SnapshotDeletionsInProgress and SnapshotInProgress that "just" return snapshots names. Maybe we could align all toString() so that they return the snapshot names and ids?

Reply:
Fair point => adjusted it :) 

Now serialises as e.g. 

```
RestoreInProgress[{oSDbrDUXT5qWCisyt_45ug}{test-restore-snapshot-repo:test-restore-snapshot2/-44f77NdQL-xY9m38VKQfw},{ZBifi65TSSqvAqbrJDPBQg}{test-restore-snapshot-repo:test-restore-snapshot1/bGqlfCTnQUWb7THtq_gjlQ}]
```

:)